CRM/Utils/File.php

```diff
@@ -604,6 +604,9 @@ public static function absoluteDirectory($directory, $basePath = NULL) {
     // make everything absolute from the baseFilePath
     $basePath = ($basePath === NULL) ? self::baseFilePath() : $basePath;
 
+    // ensure that $basePath has a trailing slash
+    $basePath = (substr($basePath, -strlen(DIRECTORY_SEPARATOR)) != DIRECTORY_SEPARATOR) ? $basePath . DIRECTORY_SEPARATOR : $basePath;
```

Comment:
@monishdeb yes but great catch finding the function in the first place :) I've added a commit and tested locally and this worked for me.